src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -325,7 +325,7 @@ case class Hdfs(properties: Hdfs.Properties) {
   val kind = "HDFS"
 }
 object Hdfs {
-  case class Properties(content: Content)
+  case class Properties(content: Content, deleted: Seq[String] = Nil, appended: Seq[String] = Nil)
```

Comment:
Should we name these `appendedFiles` and `deletedFile` (also `appendedFiles` first)?

Can we also comment what they are since it has some semantic meaning? (e.g., `appendedFiles` are the files appended since the last time `derivedDataset` is updated?

Reply:
thanks @imback82 , updated as suggested